docs/docs-content/clusters/cluster-management/platform-settings/pause-platform-upgrades.md

```diff
@@ -0,0 +1,138 @@
+---
+sidebar_label: "Pause Platform Upgrades"
+title: "Pause Platform Upgrades"
+description: "Learn about Palette's Pause Agent Upgrades setting."
+hide_table_of_contents: false
+sidebar_position: 0
+tags: ["clusters", "cluster management"]
+---
+
+Palette supports the **Pause Agent Upgrades** feature to exclude a cluster or a group of clusters from getting automatically upgraded when Palette is upgraded. The three ways to activate this feature are:
+
+* Pause Upgrades for a Single Cluster
+* Pause Upgrades for all Clusters within Project Scope
+* Pause Upgrades for all Clusters within Tenant Scope
+
+## Prerequisites
+
+<Tabs groupId="pauseUpgrades">
+
+<TabItem value="singleCluster" label="Pause Upgrades for a Single Cluster">
+
+* User access
+
+* An existing cluster. Follow this [tutorial](../../clusters.md) to get started.
+
+</TabItem>
+
+<TabItem value="projectScope" label="Pause Upgrades for all Clusters within Project Scope">
+
+* Tenant admin access
+
+* An existing cluster. Follow this [tutorial](../../clusters.md) to get started.
+
+</TabItem>
+
+<TabItem value="tenantScope" label="Pause Upgrades for all Clusters within Tenant Scope">
```

Comment:
```suggestion
```

Let's remove the tabs for the prereqs. They too similar, so we can wordsmith the permissions sentence.